Makefile

```diff
@@ -48,6 +48,18 @@ formatf:
 formatc:
 	./tools/ormolu.sh -c
 
+.PHONY: add-license
+add-license:
+	# For any Haskell or Rust file that doesn't mention AGPL yet,
+	#   add a license header.
+	# It's your own reponsibility to keep ormolu happy.
+	for file in $$(git grep -L "GNU Affero General Public License" | grep '\.hs$$\|\.rs$$'); do \
+		echo "Adding license to $${file}."; \
+		licensure -i $${file}; \
+	done;
+	@echo ""
+	@echo "you most probably want to run 'make formatf' now to keep ormolu happy"
```

Comment:
Of course this task could directly run ormolu, but I think it's preferrable to leave people the option to check their changes before having ormolu wreak havoc (potentially). So I prefer to separate these steps out and just do an echo here.